src/ClientImpl.cpp

```diff
@@ -1255,6 +1282,21 @@ void ClientImpl::run() throw (Exception, NoConnectionsException, LibEventExcepti
     m_loopBreakRequested = false;
 }
 
+static void interrupt_callback(evutil_socket_t fd, short events, void *clientData);
```

Comment:
It looks like you added a new static function, but I don't see its definition anywhere?  How does that work?  I don't get it.

Reply:
Oh, it's further down in the file.